src/client/pydaos/raw/daos_api.py

```diff
@@ -1374,12 +1375,18 @@ def __init__(self, context, cuuid=None, poh=None, coh=None):
         self.coh = ctypes.c_uint64(0)
         self.poh = ctypes.c_uint64(0)
         self.info = daos_cref.ContInfo()
+        self.cont_prop = None
+        self.cont_type = None
+        self.flag = None
 
     def get_uuid_str(self):
         """Return C representation of Python string."""
         return conversion.c_uuid_to_str(self.uuid)
 
-    def create(self, poh, con_uuid=None, cb_func=None):
+    def create(self, poh, con_uuid=None, layer_type=None,
+               enable_chksum=False, srv_verify=False,
+               chksum_type=None, chunk_size=None,
+               cb_func=None):
```

Comment:
I don't think that it is going to be practical to add all the properties here. I would be better to define a new python data structure for all those options so that it can be easily extended in the future.

Reply:
Good point. Let me think about it..

Reply:
Done. Added a structure and provided get_cont_prop and set_cont_prop. In this way, it is easy to add new values the structure as we go forward. This limits number of input parameters. Thanks Johann..